custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html

```diff
@@ -22,9 +22,12 @@
           <span class="font-16 font-montserrat font-weight-600 mr-3">
             {$ $ctrl.location_type === 'Awc' ? $ctrl.location_type.toUpperCase() : $ctrl.location_type $}s Ranking
           </span>
+          {% comment %}
+          Hiding info button
```

Comment:
Any reason not to just delete this?

Reply:
We can delete it. Just in case if they might want it back in future

Reply:
yeah let's just delete it, easy enough to add back

Reply:
done